docs/concepts/credentials/1_ctypes.md

```diff
@@ -39,6 +39,21 @@ When making a claim for a CType, all the following properties are required:
 }
 ```
 
+The CType schema is afterwards wrapped into the full CType object:
+
+```js title="CType example"
+{
+  schema:  {...} // as defined in the example above>,
+  owner: null, // DID of the owner, or null
+  hash: '0xda3861a45e0197f3ca145c2c209f9126e5053fas503e459af4255cf8011d51010' // For looking up on-chain
+}
+```
+
+## Nested CTypes
```

Comment:
After reading this section I'm not sure how this nesting works.
My expectation was that you just have another CType in your CType. So you would always have all properties and the full structure nested in the outer CType.
But the referencing looks different. You only nest fields of the CType in the new CType? Maybe this could be made clearer?

I would maybe also add a few diagrams that do some eli5.
Overall I would even go so far as to replace all json examples with a higher level abstraction like a box with a few fields in there.

Reply:
ok, I will try to explain it better. And yes, the nesting happens by reference